build.sh

```diff
@@ -31,26 +38,32 @@ install_via_pacman() {
 
 # check if virtualenv already exists
 if [ -d "$UBPORTSDOCSENV" ]; then
-  echo -e "${GREEN}Build environment found in \"$UBPORTSDOCSENV\".${PLAIN}"
+  echo -e "${GREEN}Activating virtual build environment found in \"$UBPORTSDOCSENV\".${PLAIN}"
   source $UBPORTSDOCSENV/bin/activate
 else
   echo -e "${YELLOW}Build environment not found in \"$UBPORTSDOCSENV\" ... creating it.${PLAIN}"
   echo -e "${YELLOW}Installing pip and virtualenv (using sudo).${PLAIN}"
-  # Check for apk or pacman
+  # Check for apt or pacman
   if command -v apk &> /dev/null; then
-      install_via_apk
+      echo "Detected apk"
+      sudo apk add python3 py3-pip py3-virtualenv
```

Comment:
Hmm if you're going to inline a code here, then remove the functions above? (Or vice-versa?)